blazor/datagrid/dialog-editing.md

```diff
@@ -136,4 +136,101 @@ In the below example we have changed the dialog's header text and footer button
 }
 ```
 
-> You can refer to our [Blazor DataGrid](https://www.syncfusion.com/blazor-components/blazor-datagrid) feature tour page for its groundbreaking feature representations. You can also explore our [Blazor DataGrid example](https://blazor.syncfusion.com/demos/datagrid/overview?theme=bootstrap4) to understand how to present and manipulate data.
\ No newline at end of file
+> You can refer to our [Blazor DataGrid](https://www.syncfusion.com/blazor-components/blazor-datagrid) feature tour page for its groundbreaking feature representations. You can also explore our [Blazor DataGrid example](https://blazor.syncfusion.com/demos/datagrid/overview?theme=bootstrap4) to understand how to present and manipulate data.
+
+## Implement calculated column inside grid dialog editing
+
+You can perform calculations based on the field values available in the grid's dialog editform and display the result in another field. 
```

Comment:
You can perform calculations based on the field values available in the grid's dialog edit form and display the results in another field.